benchmark/runner/mini_runners.cpp

```diff
@@ -640,6 +769,23 @@ class MiniRunners : public benchmark::Fixture {
     return plan;
   }
 
+  std::unique_ptr<planner::AbstractPlanNode> IndexNLJoinCorrector(
```

Comment:
nit: Maybe rename this to `IndexNLJoinChecker`? When you say corrector I thought this is actually going to fix something in the plan node. But this is just correctness checking, right? Similar with the function below.